tests/baselines/reference/docker/vscode.log

```diff
@@ -65,6 +65,7 @@ $ gulp compile --max_old_space_size=4095
   Type 'null' is not assignable to type 'string'.
 [XX:XX:XX] Error: /vscode/src/vs/editor/contrib/rename/renameInputField.ts(93,3): Type 'string | null' is not assignable to type 'string'.
   Type 'null' is not assignable to type 'string'.
+[XX:XX:XX] Error: /vscode/src/vs/editor/contrib/codeAction/codeActionCommands.ts(118,30): Expected 3 arguments, but got 2.
```

Comment:
Already fixed in [code itself](https://github.com/microsoft/vscode/commit/4b6e4ee176acba945da27405b41712999f46f4c9)